src/components/Pega_Extensions_JapaneseInput/index.tsx

```diff
@@ -0,0 +1,208 @@
+import { useEffect, useState, useRef } from 'react';
+import { Input, FieldValueList, Text, withConfiguration } from '@pega/cosmos-react-core';
+import '../create-nonce';
+
+// interface for props
+interface PegaExtensionsJapaneseInputProps {
```

Comment:
extend InputProps from @pega/cosmos-react-core. In that case, you can remove duplicate types.